portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/controlpanel/apps/appmanager/AppManagement.testcase

```diff
@@ -559,6 +559,92 @@ definition {
 			appStatus = "Active");
 	}
 
+	@description = "This test asserts that user can filter app by Category in marketplace store."
+	@priority = "3"
+	test CanFilterMarketplaceStoreAppsByCategories {
+		property test.name.skip.portal.instance = "AppManagement#CanFilterMarketplaceStoreAppsByCategories";
+
+		task ("Given user login marketplace") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			AppManager.loginMarketplaceStore();
+
+			Navigator.openURL();
+		}
+
+		task ("When user filter by Category with value Page Layouts") {
+
+			// Navigate the store again for filter price failure issue
+
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			WaitForPageLoad();
+
+			AppManager.filtersApp(
+				filterBy = "    Page Layouts ",
+				filterName = "Category");
+		}
+
+		task ("Then the results filtered by Page Layouts display properly") {
+			AssertElementPresent(
+				key_appCategory = "Page Layouts",
+				locator1 = "AppManager#APP_CATEGORY");
+
+			AssertElementNotPresent(
+				key_appCategory = "Themes / Site Templates",
+				locator1 = "AppManager#APP_CATEGORY");
+		}
+	}
+
+	@description = "This test asserts that user can filter app by liferay version in marketplace store."
+	@priority = "3"
+	test CanFilterMarketplaceStoreAppsByLiferayVersion {
+		property test.name.skip.portal.instance = "AppManagement#CanFilterMarketplaceStoreAppsByLiferayVersion";
+
+		task ("Given user login marketplace") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Marketplace",
+				panel = "Control Panel",
+				portlet = "Store");
+
+			AppManager.loginMarketplaceStore();
+
+			Navigator.openURL();
+		}
+
+		task ("When user filter by Liferay Version with value Liferay Portal 6.1 CE GA3") {
+
+			// Navigate the store again for filter price failure issue
```

Comment:
Make sure this step is necessary and remember to update the description accordingly.

Reply:
done